src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -272,27 +271,22 @@ function createOpenTagMarkup(
     if (propValue == null) {
       continue;
     }
-    if (!registrationNameModules.hasOwnProperty(propKey)) {
-      if (propKey === STYLE) {
-        propValue = createMarkupForStyles(propValue, instForDebug);
-      }
-      var markup = null;
-      if (isCustomComponent(tagLowercase, props)) {
-        if (!RESERVED_PROPS.hasOwnProperty(propKey)) {
-          markup = DOMMarkupOperations.createMarkupForCustomAttribute(
-            propKey,
-            propValue,
-          );
-        }
-      } else {
-        markup = DOMMarkupOperations.createMarkupForProperty(
+    if (propKey === STYLE) {
+      propValue = createMarkupForStyles(propValue, instForDebug);
+    }
+    var markup = null;
+    if (isCustomComponent(tagLowercase, props)) {
```

Comment:
I wonder what happens if you pass `onChange` to a web component. Hmmm. I'll investigate.